index.html

```diff
@@ -237,8 +244,11 @@ <h2>Securing JSON-LD Verifiable Credentials with JOSE</h2>
           to [[VC-DATA-MODEL-2.0]].
         </p>
         <p>
-          [[RFC7515]] MAY be used to secure this media type.
-          The <code>typ</code> header parameter SHOULD be <code>vc+ld+json+sd-jwt</code>.
+          JWS [[RFC7515]] is used to secure this media type with JOSE.
```

Comment:
should we add a `MUST` here?

Reply:
No need.  It means the same thing with and without the MUST.  Plain English text is just as normative as RFC 2119 language.  And it reads more naturally as is.